build.rs

```diff
@@ -35,19 +35,29 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     fs_extra::copy_items(&source_paths, &out_dir, &copy_options)?;
 
     // Compile the C library.
-    let mut make = Command::new("make");
-
-    make.env_remove("PROFILE").arg("-C").arg(&out_dir).arg("build");
-
-    if env::var("PROFILE").unwrap() == "debug" {
-        make.arg("DEBUG=1");
+    let mut build = cc::Build::new();
+    build
+        .files(glob(out_dir.join("src/*.c").to_str().unwrap()).unwrap().map(|p| p.unwrap()))
+        .files(glob(out_dir.join("src/postgres/*.c").to_str().unwrap()).unwrap().map(|p| p.unwrap()))
+        .file(out_dir.join("vendor/protobuf-c/protobuf-c.c"))
+        .file(out_dir.join("vendor/xxhash/xxhash.c"))
+        .file(out_dir.join("protobuf/pg_query.pb-c.c"))
+        .include(out_dir.join("."))
+        .include(out_dir.join("./vendor"))
+        .include(out_dir.join("./src/postgres/include"))
+        .include(out_dir.join("./src/include"));
+    if env::var("PROFILE").unwrap() == "debug" || env::var("DEBUG").unwrap() == "1" {
+        build.debug(true);
+        build.opt_level(0);
+        build.define("USE_ASSERT_CHECKING", None);
```

Comment:
We don't actually have to manually call `debug` or `opt_level` here - these are automatically set by `cc` based on the values of `OPT_LEVEL` and `DEBUG`, which are [set automatically](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts) by cargo when running a build script.

We can however still rely on `DEBUG` here for `USE_ASSERT_CHECKING`